src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -74,46 +74,22 @@
 }
 
 .contactmoment {
-  &__details {
-    // Set table width for word-break
-    table {
-      margin-top: 0;
-      overflow-x: auto;
-      table-layout: fixed;
-      width: 100%;
-      @media (min-width: 768px) {
-        width: var(--mobile-ms-width);
-      }
-    }
-    table th {
-      width: 100px;
-      white-space: nowrap;
-    }
-    table td {
-      width: 300px;
-      overflow: hidden;
-      word-break: break-word;
-      white-space: normal;
-
-      &.table__item--status {
-        .table__item--wrapper {
-          display: flex;
-          justify-content: space-between;
-        }
-      }
+  &__heading {
+    font-family: var(--font-family-heading);
+    font-size: var(--font-size-heading-3);
+    font-weight: var(--font-weight-heading-3);
+    line-height: var(--font-line-height-heading-3);
+    margin: 0;
```

Comment:
Nice margin reset :-) but... in the design you can see you will still need a bottom margin for Headings (var(--spacing-large)).